src/HexEdit.jl

```diff
@@ -106,12 +106,11 @@ function dump!(s::HexEd, start = nothing, n = nothing)
 end # function dump!
 
 #----------
-# converts ASCII string or hexadecimal string to binary byte
-# array
+# converts ASCII string or hexadecimal string to binary byte array
 #----------
 function hex2bin(rawstr::AbstractString)
-    if (!ismatch(r"^0x[0-9a-fA-F]+", rawstr))
-        return convert(Array{UInt8}, rawstr)
+    if (match(r"^0x[0-9a-fA-F]+", rawstr) == nothing)  # 如果不是 16进制数的 普通字符串
```

Comment:
If an informative comment is necessary here, it would be nice to have it in English.

Also, this can be
```julia
if occursin(r"^0x[0-9a-fA-F]+", rawstr)
```
since the `RegexMatch` object constructed by calling `match` is never used.